src/providers/ChatAppProvider.tsx

```diff
@@ -447,6 +448,26 @@ const ChatAppContextProvider: React.FC<ChatAppContextProviderProps> = ({
 
       discardMessageDraft();
     }
+    if(file){
+      await kitty.sendMessage({
+        channel: channel,
+        file: file,
```

Comment:
syntax nit: this could just be `file,` or `channel,` in shorthand.

Reply:
removed the unnecessary part.